clutter/SplitScreenScanner
# Update scanner style guide

## Pivotal Ticket
https://www.pivotaltracker.com/story/show/158335530

## Feature Description
SplitScreenScanner has been updated with a new light theme!

![img_0111 2](https://user-images.githubusercontent.com/10718632/41441698-9c0d030c-6fe8-11e8-8ccf-b301921dabe2.PNG)

![img_0110 2](https://user-images.githubusercontent.com/10718632/41441721-afb77ff4-6fe8-11e8-9997-f83586e7fd16.PNG)

![img_0112 2](https://user-images.githubusercontent.com/10718632/41441703-a3488baa-6fe8-11e8-9cc7-f34019101897.PNG)

![img_0108 2](https://user-images.githubusercontent.com/10718632/41441710-a815ef60-6fe8-11e8-8932-cb623dfc5aef.PNG)

![img_0109 2](https://user-images.githubusercontent.com/10718632/41441727-b5756500-6fe8-11e8-820b-8fd796467a96.PNG)


## How to Test?
- Open up the test app and make sure all screens adhere to the new light theme (make sure there are no traces of the old dark theme left)
